stackslib/src/net/download/nakamoto/tenure_downloader_set.rs

```diff
@@ -407,19 +482,43 @@ impl NakamotoTenureDownloaderSet {
                 continue;
             };
 
+            let attempt_count = if let Some(attempt_count) = self.attempted_tenures.get(&ch) {
+                *attempt_count
+            } else {
+                0
+            };
+            self.attempted_tenures
+                .insert(ch.clone(), attempt_count.saturating_add(1));
+
+            let attempt_failed_count =
+                if let Some(attempt_failed_count) = self.attempt_failed_tenures.get(&ch) {
+                    *attempt_failed_count
+                } else {
+                    0
+                };
+
             info!("Download tenure {}", &ch;
+                "peer" => %naddr,
+                "attempt" => attempt_count.saturating_add(1),
+                "failed" => attempt_failed_count,
+                "downloads_scheduled" => %self.num_scheduled_downloaders(),
+                "downloads_total" => %self.num_downloaders(),
+                "downloads_max_count" => count,
+                "downloads_inflight" => self.inflight(),
                 "tenure_start_block" => %tenure_info.start_block_id,
                 "tenure_end_block" => %tenure_info.end_block_id,
                 "tenure_start_reward_cycle" => tenure_info.start_reward_cycle,
-                "tenure_end_reward_cycle" => tenure_info.end_reward_cycle);
+                "tenure_end_reward_cycle" => tenure_info.end_reward_cycle,
+                "tenure_burn_height" => tenure_info.tenure_id_burn_block_height);
 
             debug!(
-                "Download tenure {} (start={}, end={}) (rc {},{})",
+                "Download tenure {} (start={}, end={}) (rc {},{}) burn_height {}",
                 &ch,
                 &tenure_info.start_block_id,
                 &tenure_info.end_block_id,
                 tenure_info.start_reward_cycle,
-                tenure_info.end_reward_cycle
+                tenure_info.end_reward_cycle,
+                tenure_info.tenure_id_burn_block_height,
             );
```

Comment:
This debug log looks like it just duplicates a bunch of info from the info log. Only one log line seems necessary here.

Reply:
Sure, will drop the debug